### PR TITLE
Add github actions for tests

### PR DESCRIPTION
**Pull request by jvargas98**

### CODE REVIEW

File: .github/workflows/ruby.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:12
+        ports: ['5432:5432']
+        env:
+          POSTGRES_USER: clockapp
+          POSTGRES_PASSWORD: password
```
Comment (OmauryMarmolejo) [inline .github/workflows/ruby.yml]: Please try to protect secrets

File: .github/workflows/ruby.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            ${{ runner.os }}-yarn-
+      - name: Install packages
+        run: |
+          yarn install --pure-lockfile
+      - name: Setup test database
+        env:
+          RAILS_ENV: test
+          PG_HOST: localhost
+          PG_USER: clockapp
+          PG_PASSWORD: password
```
Comment (OmauryMarmolejo) [inline .github/workflows/ruby.yml]: Same here

File: .github/workflows/ruby.yml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        run: |
+          bin/rails db:create
+          bin/rails db:migrate
+      - name: Run Tests
+        env:
+          PG_HOST: localhost
+          PG_USER: clockapp
+          PG_PASSWORD: password
```
Comment (OmauryMarmolejo) [inline .github/workflows/ruby.yml]: Same here

File: .github/workflows/ruby.yml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+          - 6379:6379
+        options: --entrypoint redis-server
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby
+        # To automatically get bug fixes and new Ruby versions for ruby/setup-ruby,
+        # change this to (see https://github.com/ruby/setup-ruby#versioning):
+        uses: ruby/setup-ruby@v1
+        #uses: ruby/setup-ruby@473e4d8fe5dd94ee328fdfca9f8c9c7afc9dae5e
```
Comment (OmauryMarmolejo) [inline .github/workflows/ruby.yml]: Please remove this line if you're not using it

File: .github/workflows/ruby.yml (hunk 1 above)
Comment (jvargas98) [reply .github/workflows/ruby.yml]: It's done, thanks for the comment ❤️

File: .github/workflows/ruby.yml (hunk 2 above)
Comment (jvargas98) [reply .github/workflows/ruby.yml]: already changed, thanks

File: .github/workflows/ruby.yml (hunk 4 above)
Comment (jvargas98) [reply .github/workflows/ruby.yml]: already removed, thanks

File: .github/workflows/ruby.yml (hunk 3 above)
Comment (jvargas98) [reply .github/workflows/ruby.yml]: has been changed as well

**Pull request merged**